pyquil/tests/test_qvm_client.py

```diff
@@ -0,0 +1,160 @@
+##############################################################################
```

Comment:
Note: don't forget to add this header to all the new files

Reply:
Good catch. I thought I got them all, but I had missed a couple.

Reply:
There's probably a tool we could use in the future for this.